polyfill/lib/calendar.mjs

```diff
@@ -68,6 +89,26 @@ export class Calendar {
     void options;
     throw new Error('not implemented');
   }
+  timeAdd(time, duration, options, constructor) {
+    void time;
+    void duration;
+    void options;
+    void constructor;
+    throw new Error('not implemented');
+  }
+  timeSubtract(time, duration, options, constructor) {
```

Comment:
Related: #1066 (for dates). But if we're adding time methods for the first time, why not just have one?

Reply:
The original calendar draft had only one add() and one subtract() method. I can't remember exactly, but it may not be possible to combine dateAdd() and timeAdd() since the allowed ranges for all the Plain types are different. But it may be possible to keep that range checking in the callers.